flag.go

```diff
@@ -31,12 +32,19 @@ func printHelp() {
 // parseCliArgs parses CLI options and sets several important global variables. Call it early.
 func parseCliArgs() {
 	var createAdminName string
+	var versionFlag bool
 
 	flag.StringVar(&cfg.ListenAddr, "listen-addr", "", "Address to listen on. For example, 127.0.0.1:1737 or /run/mycorrhiza.sock.")
 	flag.StringVar(&createAdminName, "create-admin", "", "Create a new admin. The password will be prompted in the terminal.")
+	flag.BoolVar(&versionFlag, "version", false, "Print the mycorrhiza version and exit.")
 	flag.Usage = printHelp
 	flag.Parse()
 
+	if versionFlag {
+		fmt.Println("mycorrhiza:", version.TaggedRelease, "\tcommit:", version.CommitHash)
```

Comment:
IMO it would be better to print something like:

`Mycorrhiza Wiki v1.11.0+dbaf874`

or for dirty working trees:

`Mycorrhiza Wiki v1.11.0+dbaf874-dirty`

or for releases:

`Mycorrhiza Wiki v1.11.0`

Reply:
I probably spent an hour trying to figure out how to format that version information haha. You're suggestion is definitely a lot better, thanks!